python/lsst/daf/butler/registry/dimensions/static.py

```diff
@@ -50,6 +52,33 @@
 _VERSION = VersionTuple(5, 0, 0)
 
 
+def _makeDimensionGraphTableSpec() -> ddl.TableSpec:
+    """Create a specification for a table that holds `DimensionGraph`
+    definitions.
+
+    Returns
+    -------
+    spec : `ddl.TableSpec`
+        Specification for the table.
+    """
+    return ddl.TableSpec(
+        fields=[
+            ddl.FieldSpec(
+                name="digest",
+                dtype=sqlalchemy.String,
+                length=DimensionGraph.DIGEST_SIZE,
+                primaryKey=True,
+            ),
+            ddl.FieldSpec(
+                name="dimension_name",
+                dtype=sqlalchemy.String,
+                length=64,
```

Comment:
We should have probably defined a bunch of constants for these `64` magic numbers too.

Reply:
Are we sure they don't get converted to TEXT...?

Reply:
At this point I'd be more inclined to just use `sqlalchemy.Text` and leave the length blank. So many of them are arbitrary, or at best "guess and check".

Reply:
(messages crossed) I think this does get translated to TEXT, but that's controlled by different code, so the magic number here is still a bit problematic.

Reply:
My main comment is to note that if you needed it to be exactly 64 then it probably isn't happening. The threshold is >32 for auto conversion to TEXT. You can use TEXT explicitly in the type if you want it to be TEXT. You can't though require a long varchar to actually be a varchar.